usr/sbin/p11sak/p11sak.c

```diff
@@ -538,6 +550,9 @@ static void print_gen_ec_help(void)
             "      --label LABEL                           key label LABEL to be listed\n");
     printf(
             "      --attr [S D G U X]                      set key attributes\n");
+    printf( "      -c, --custom-attr [[pub_attrs]:[priv_attrs]] \n");
+    printf( "             for asymmetric keys: set individual key attributes, for more info type:\n");
+    printf( "             man p11sak\n");
```

Comment:
....for more info, see the p11sak man page.

Reply:
Changed to see man 1 p11sak